views/game.hbs

```diff
@@ -2,30 +2,49 @@
 <html>
 
 <head>
-  <meta charset="utf-8" />
+  <meta charset="utf-8" {{#falseOrFalse readyToPlay currentPlayerTurnToPlay}}http-equiv="refresh" content="5"
+    {{/falseOrFalse}} />
   <title>Hex</title>
   <link rel="stylesheet" type="text/css" href="/css/style.css" />
   <link rel="stylesheet" type="text/css" href="/css/game.css" />
 </head>
 
 <body>
-  {{#areEqual state.winner null}}
+  {{#areEqual game.state.winner null}}
   <div class="gameboard">
     <div class="box-current-player">
-      <p>Current player :</p>
-      <p class="current-player" data-player="{{state.turn}}">{{state.turn}}</p>
+      {{#areEqual readyToPlay true}}
+      <p>
+        Current player:
+        <span class="current-player" data-player="{{game.state.turn}}">{{game.state.turn}}</span>
+      </p>
+      {{#areEqual currentPlayerTurnToPlay false}}
+      <p>Waiting for other player to make their move...</p>
+      {{/areEqual}}
```

Comment:
indentation is weird

Reply:
I agree but it's fixed by the default conf of prettier..